recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/doc/local.conf.dom0-image-minimal-initramfs

```diff
@@ -225,6 +225,9 @@ DISTRO_FEATURES_append = " xen"
 # Configuration for Para-virtual camera
 DISTRO_FEATURES_append = " pvcamera"
 
+# Configuration for VirtIO feature
+DISTRO_FEATURES_append = " virtio"
```

Comment:
maybe it is reasonable to make an override for the machines to support the ''H3 ES3" ?